tests/drivers/flash/erase_blocks/README.rst

```diff
@@ -0,0 +1,6 @@
+Flash Controller Erase Block Testing
+#####################
```

Comment:
The `##########` should match the length of the above line

Reply:
> The `##########` should match the length of the above line

Updated. Thank you.